skytap/resource_skytap_vm.go

```diff
@@ -115,6 +115,16 @@ func resourceSkytapVM() *schema.Resource {
 					},
 				},
 			},
+			"external_ips": {
```

Comment:
Double check if we  can have only a string, as all the  ips will be the same given a VM